c++/Makefile.am

```diff
@@ -481,7 +481,7 @@ capnp_test_LDADD = libcapnp-test.a libcapnp.la libkj-test.la libkj.la
 
 else !LITE_MODE
 
-check_PROGRAMS = capnp-test capnp-evolution-test capnp-afl-testcase
+check_PROGRAMS = capnp-test capnp-evolution-test capnp-afl-testcase fuzz-capnp-run
```

Comment:
`fuzz-capnp-run` seems like a strange name to me. Maybe `capnp-llvm-fuzzer-testcase`?

Reply:
ok